dist/tools/headerguards/headerguards.py

```diff
@@ -98,6 +98,10 @@ def fix_headerguard(filename):
         print("%s: #pragma once and classic header guards used in the same file" %
               filename, file=sys.stderr)
         return False
+    elif (pragma_once_found > 1):
+        print("%s: More than one #pragma once in the same file" %
+              filename, file=sys.stderr)
```

Comment:
Does not have to be for this PR, but
```suggestion
        print(f"{filename}: More than one #pragma once in the same file",
              file=sys.stderr)
```

in almost any case is more performant (and I don't think we do still run python versions that do not support f-strings).